CheckpointPlugin.cpp

```diff
@@ -135,7 +135,7 @@ void CheckpointPlugin::onLoad()
 	cvarManager->registerCvar("enable_roll_unpause", "1", "Enable roll to unpause", true, true, 0, true, 1, true);
 	cvarManager->registerCvar("roll_threshold", "0.1", "Threshold for roll to unpause", true, true, 0, true, 1, true);
 
-	cvarManager->registerCvar("rewind_threshold", "0.2", "Threshold for steering to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("rewind_threshold", "0.05", "Threshold for steering to unpause", true, true, 0, true, 1, true);
```

Comment:
Looks like the help text here is a copy/paste-o?

Reply:
yup, whoops